database/migrations/2018_09_09_111023_create_dev_rant_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateDevRantTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('devRant_community', function (Blueprint $table) {
+            $table->increments('id');
+            $table->foreign('id')->references('id')->on('users');
+            $table->integer('devRant_userID');
```

Comment:
ID should be lowercase as well as devRant

Reply:
fixed this, too

Reply:
Still uppercase characters in here?